src/isomorphic/classic/types/__tests__/ReactPropTypes-test.js

```diff
@@ -466,6 +466,11 @@ describe('ReactPropTypes', function() {
     it('should accept empty array for required props', function() {
       typeCheckPass(PropTypes.node.isRequired, []);
     });
+
+    it('should accept both single child and array of children for the children prop type', function() {
+      typeCheckPass(PropTypes.children, <div/>);
+      typeCheckPass(PropTypes.children, <div><div/></div>);
```

Comment:
FTR: This is also a single child, not an array of children.  This is a single child that happens to have it's own child.